pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   plugin_platform_interface: ^2.0.2
   # plugin_platform_interface depends on meta anyway
   meta: ^1.3.0
+  stack_trace: ^1.11.1
```

Comment:
see https://github.com/PostHog/posthog-flutter/blob/b339e95e29c2f152f03a8581c101a98da6f175f4/pubspec.yaml#L10
our min dart version is currently 3.3
v1.11.1 supports Dart >= 2.18 https://pub.dev/packages/stack_trace/versions
so we'd need to use min [1.12.0](https://pub.dev/packages/stack_trace/versions/1.12.0) and bump our min version of the Dart SDK and Flutter to 3.4.0 as well to be compatible with the stack trace version